cmd/root.go

```diff
@@ -7,7 +7,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/sourcegraph/checkup"
+	"https://github.com/monosolutions/checkup"
```

Comment:
Ummm... I assume this is an accident? :)

Reply:
yes